.github/workflows/build-test.yml

```diff
@@ -765,15 +777,34 @@ jobs:
         command: install
         args: --git https://github.com/rustwasm/twiggy.git --rev ${{ steps.twiggy-version.outputs.hash }} twiggy
 
+    - name: Install npm dependencies from cache
+      uses: actions/cache@v2
+      id: npm-cache
+      with:
+        path: ffi/diplomat/wasm/node_modules
+        key: npm-${{ hashFiles('ffi/diplomat/wasm/package-lock.json') }}
+
+    - name: Install npm packages.
+      # If npm dependencies are not found in cache.
+      if: steps.npm-cache.outputs.cache-hit != 'true'
+      # Note: if package.json and package-lock.json are out of sync then
+      # 'npm ci' will fail. Locally execute 'npm install' in directory
+      # 'ffi/diplomat/wasm' to generate a new package-lock.json file and
+      # merge it into the repository.
+      run: |
+        cd ffi/diplomat/wasm
+        npm ci
```

Comment:
Make failures of `npm ci` print error message to user on how to fix, in line with above comments.  But still remember to preserve failing exit code.

Reply:
I don't know if this is needed.  It should be rare for `npm ci` to fail, since anyone who edits `package.json` needs to run `npm install` which updates `package-lock.json`, and the comments here in the file seem helpful enough.

Reply:
Added message how to fix failing npm ci, error case handling is the most explicit now, e.g. in case someone overlooks to add the new package-lock.json file.

Reply:
Looks good, thanks.  It could be useful for new contributors -- it's not always clear whether a `*.lock` file should be committed into version control.  The general wisdom is that ["you don't need to commit `.lock` files for libraries"](https://stackoverflow.com/questions/61037557/should-i-commit-lock-file-changes-separately-what-should-i-write-for-the-commi/66884399#66884399), but of course you should for applications. I think it's loosely analogous to the question of [whether to include `Gemfile.lock` for Jekyll content](https://stackoverflow.com/questions/56261127/is-the-gemfile-lock-file-needed-in-a-jekyll-site-hosted-with-github-pages) (ex: Github Pages markdown).